.gitignore

```diff
@@ -102,3 +102,5 @@ ENV/
 
 # mypy
 .mypy_cache/
+.idea/workspace.xml
```

Comment:
you'll want to include `.idea/` and that will exclude everything under that folder.  Please do that and remove the `.idea/` from your commit otherwise its going to override everyone in the master repo.

https://stackoverflow.com/questions/1143796/remove-a-file-from-a-git-repository-without-deleting-it-from-the-local-filesyste